### PR TITLE
Uv/coverage

### PR DESCRIPTION
**Pull request by uv-orbs**

### CODE REVIEW

File: models/swap_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	var frags []OrderFrag
+	for i := 0; i < 10; i++ {
+		sz := int32(1000 * (i + 1))
+		frags = append(frags, OrderFrag{
+			OrderId: orderId,
+			Size:    decimal.NewFromInt32(sz),
+		})
+	}
+
+	t.Run("happy path", func(t *testing.T) {
```
Comment (Luke-Rogerson) [inline models/swap_test.go:23]: Nice! A table test

File: models/swap_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		var expected = `[{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"1000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"2000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"3000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"4000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"5000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"6000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"7000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"8000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"9000"},{"orderId":"97522a1a-7648-4b4f-97b2-38d90a5e2bd0","size":"10000"}]`
+		res, err := MarshalOrderFrags(frags)
+		assert.NoError(t, err)
+		assert.Equal(t, string(res), expected)
+		println(string(res))
```
Comment (Luke-Rogerson) [inline models/swap_test.go:28]: Remove this?

File: service/taker_test.go [hunk 3]
```diff
@@ -19,18 +19,23 @@ func TestTaker_Quote(t *testing.T) {
...
 			Error: models.ErrSwapInvalid,
 			Sets:  make(map[string]map[string]struct{}),
 		}
+		// buy
 		svc, _ := service.New(&store, evmClient)
 		_, err := svc.GetQuote(ctx, symbol, models.BUY, decimal.Zero)
 		assert.Error(t, err, models.ErrInAmount)
+		// sell
+		svc, _ = service.New(&store, evmClient)
+		_, err = svc.GetQuote(ctx, symbol, models.SELL, decimal.Zero)
```
Comment (Luke-Rogerson) [inline service/taker_test.go:33]: I suggest you assert against the return of quote too

File: models/swap_test.go (hunk 2 above)
Comment (uv-orbs) [reply models/swap_test.go:28]: the println you mean?

**Pull request merged**